apps/etterlatte-egne-ansatte-lytter/src/main/kotlin/kafka/KafkaConsumerConfiguration.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.etterlatte.kafka
+
+import io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig
+import io.confluent.kafka.serializers.KafkaAvroDeserializer
+import io.confluent.kafka.serializers.KafkaAvroDeserializerConfig
+import org.apache.kafka.clients.CommonClientConfigs
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.common.config.SslConfigs
+import org.apache.kafka.common.security.auth.SecurityProtocol
+import org.apache.kafka.common.serialization.StringDeserializer
+import java.util.Properties
+
+interface KafkaConsumerConfiguration {
+    fun generateKafkaConsumerProperties(env: Map<String, String>): Properties
+}
+
+class KafkaEnvironment : KafkaConsumerConfiguration {
+    override fun generateKafkaConsumerProperties(env: Map<String, String>): Properties {
+        val fiveMinutesInMs = 300000
```

Comment:
```suggestion
        val maxPollInterval = Duration.of(5, ChronoUnit.MINUTES).toMillis()
```
og så rename lenger ned (evt gjøre det inline ved MAX_POLL_INTERVAL_MS_CONFIG)

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/884/commits/f3859050f542db91d25194748e9e0c16b6c957f9